src/sdk/pynni/nni/tuner.py

```diff
@@ -17,31 +17,128 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT
 # OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 # ==================================================================================================
+
+"""
+Tuner is an AutoML algorithm, which generates a new configuration for the next try.
+A new trial will run with this configuration.
+
+See :class:`Tuner`' specification and ``docs/en_US/tuners.rst`` for details.
+"""
+
 import logging
 
 import nni
 
 from .recoverable import Recoverable
 
+__all__ = ['Tuner']
+
 _logger = logging.getLogger(__name__)
 
 
 class Tuner(Recoverable):
+    """
+    Tuner is an AutoML algorithm, which generates a new configuration for the next try.
```

Comment:
generates configurations to run with.

Reply:
This sentence is copied from [overview](https://github.com/microsoft/nni/blob/master/docs/en_US/Overview.md).
Do you think it should be updated?

Reply:
Not sure. "For the next try" looks really Chi-english to me.